core/domain/question_services.py

```diff
@@ -217,3 +226,142 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+
+
+def get_new_question_id():
+    """Returns a new question id.
+
+    Returns:
+        str. A new question id.
+    """
+    return question_models.QuestionModel.get_new_id('')
+
+
+def create_question_summary(
+        question_id, creator_id):
```

Comment:
I think this will fit in previous line itself?

Reply:
Done.